addon/components/nrg-search.js

```diff
@@ -64,6 +64,9 @@ export default class NrgSearchComponent extends NrgValidationComponent {
   }
 
   get canPerformSearch() {
+    if (this.args.minCharacters === '0') {
+      return true;
+    }
     return this.searchString?.length >= this.minCharacters;
```

Comment:
```suggestion
    return (this.searchString?.length ?? 0) >= this.minCharacters;
```

Is there a need to check the arg here since it is the same value as `this.minCharacters`? I think we're also doing a strict comparison to a string with this, while the default value is set to a number. What if instead we just default the value to 0 if `this.searchString` is nullish?